rai/node/lmdb.cpp

```diff
@@ -1829,12 +1829,12 @@ void rai::mdb_store::checksum_del (rai::transaction const & transaction_a, uint6
 
 void rai::mdb_store::flush (rai::transaction const & transaction_a)
 {
-	std::unordered_map<rai::account, std::shared_ptr<rai::vote>> sequence_cache_l;
 	{
 		std::lock_guard<std::mutex> lock (cache_mutex);
-		sequence_cache_l.swap (vote_cache);
+		vote_cache_l2 = vote_cache_l1;
```

Comment:
Since we're checking both l1 and l2, do we need to copy this, it seems like a swap would still work.
At the end of the for-look that checks l2 we could clear l2.

Reply:
> Since we're checking both l1 and l2, do we need to copy this, it seems like a swap would still work.
Yeah good point.

>  At the end of the for-look that checks l2 we could clear l2.
I believe that would result in worse performance. Also, we'd need to do it after the transaction expires, not at the end of the function.